src/protocols/aeternity/composables/aeNames.ts

```diff
@@ -14,6 +14,7 @@ import type {
   IAuctionBid,
   AccountAddress,
 } from '@/types';
+import type { Name } from '@aeternity/aepp-sdk/src/utils/encoder-types';
```

Comment:
```suggestion
import { Encoding } from '@aeternity/aepp-sdk';
```
And then use `Encoding.Name`. Sdk does not guarantee that the path you have used will remain. But do guarantee that you would be able to get it from default import.

Reply:
Replaced with `Encoded.Name` in all instances